### DISCUSSION TITLE
How do you use the autocomplete feature?

### DISCUSSION THREAD

**Discussion by nitanmarcel** [LunarVim/LunarVim#2861] [Q&A] [answered]
I'm trying to use the autocompletion feature and while I get the popups with the suggestions I can't find how to select them. I tried using TAB and arrows but nothing happens.

**Comment by rebuilt** [comment, marked as answer]
`TAB` should work to cycle forwards through the suggestions.  Then `Enter` should insert the selected option into your file.   If that doesn't work, check your keymappings to make sure nothing else is overriding the TAB mapping.   

`:verbose imap <TAB>` should output `<Cmd>call v:lua.cmp.utils.keymap.set_map(8)<CR>`

This means that the cmp plugin is being used when you press TAB

**Reply by nitanmarcel** [reply]
It outputs this.

![image](https://user-images.githubusercontent.com/41646249/181288174-b39a1fe3-7c23-44d7-a261-683e4aed5ec9.png)

**Reply by nitanmarcel** [reply]
Fixed it. 

When I generated the config.lua file it overriden the function. Removing `lvim.keys.normal_mode["<C-s>"] = ":w<cr>"
` fixed the problem

**Comment by hrai** [comment]
Is there a way to select the first item on <CR> press without using any tabs? @rebuilt
